### DISCUSSION TITLE
what should i do ,when i use useDraggable ，in some condition, i want control stop the dragging

### DISCUSSION THREAD

**Discussion by luqiudi** [vueuse/vueuse#1110] [Q&A]
what should i do ,when i use useDraggable ，in some condition, i want control stop the dragging.
then  in some condition ,i want resume the draggable.

situation:
i want drag an element in his parent element ,what should i do

**Comment by wheatjs** [comment]
Can you expand a bit more on what you want to do? Are you looking for an option to enable and disable dragging?

**Reply by velly** [reply]
Is there anyway to stop moving when the postion x or y become negative?!

^9.13.0

**Comment by velly** [comment]
I can stop it anywhere using renderless component.

const position = ref<Position>({x,y);

useDraggable(dragRoot, {
  initialValue: {
    x: 300,
    y: 300,
  },
  preventDefault: true,
  handle: topicTree,
  onMove({ x, y }) {
    if (x > 0 && y > 0 && display.width.value - x >= 120) {
      position.value = { x, y };
    }
  },
});

const styles = computed(() => {
  return {
    left: `${position.value.x}px`,
    top: `${position.value.y}px`,
  };
})

bindint the styles on root draggable.
You can do everything you want.
